pkg/api/resource/resource_helpers.go

```diff
@@ -143,10 +113,10 @@ func GetNodeCondition(status *NodeStatus, conditionType NodeConditionType) (int,
 	return -1, nil
 }
 
-// Updates existing pod condition or creates a new one. Sets LastTransitionTime to now if the
+// UpdatePodCondition updates existing pod condition or creates a new one. Sets LastTransitionTime to now if the
```

Comment:
I moved the pod utilities to pkg/api/pod, and removed the unused node utilities. There are two remaining functions that are related to "resource" in this file. PTAL. Thanks, @lavalamp :)